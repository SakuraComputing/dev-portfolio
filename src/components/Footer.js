import React from 'react';

export const Footer = () => (
    <footer className="footer">
        <div className="container text-center">
            <small className="copyright">Designed with <i className="fa fa-heart"></i> by <a
                href="/" target="_blank">Xiaoying Riley</a> for developers
            </small>
        </div>
    </footer>
);

export default Footer;