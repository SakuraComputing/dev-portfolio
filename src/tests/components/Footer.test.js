import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

describe('Footer Section', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });
});