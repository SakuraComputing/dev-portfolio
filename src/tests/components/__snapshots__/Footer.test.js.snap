// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Section should render correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container text-center"
  >
    <small
      className="copyright"
    >
      Designed with 
      <i
        className="fa fa-heart"
      />
       by 
      <a
        href="http://themes.3rdwavemedia.com"
        target="_blank"
      >
        Xiaoying Riley
      </a>
       for developers
    </small>
  </div>
</footer>
`;
